app/build.gradle

```diff
@@ -26,6 +26,15 @@ android {
         exclude 'META-INF/NOTICE.txt'
         exclude 'AndroidManifest.xml'
     }
+
+    buildTypes {
```

Comment:
why is there a beta in a lib?

Reply:
To be able to test "release" builds without marking them as release.

Reply:
Ok, I will remove it.